lib/flipper/api/v1/decorators/feature.rb

```diff
@@ -0,0 +1,33 @@
+require 'delegate'
+require 'flipper/api/v1/decorators/gate'
+
+module Flipper
+  module Api
+    module V1
+      module Decorators
+        class Feature < SimpleDelegator
+
+          # Public: The feature being decorated.
+          alias_method :feature, :__getobj__
+          
+          # Public: Returns instance as hash that is ready to be json dumped.
+          def as_json
+            {
```

Comment:
Oh, actually I just noticed you included id as name.to_s. I missed that. I think I'd rather see key => key than id. What was your reasoning for using id for key?